VMEncryption/main/Utils/HandlerUtil.py

```diff
@@ -418,6 +418,13 @@ def redo_last_status(self):
             previous_seq = str(latest_sequence_num - 1)
             previous_status_file = os.path.join(self._context._status_dir, previous_seq + '.status')
 
+            if not os.path.exists(previous_status_file):
```

Comment:
Not sure about the scenarios here. The guest agent telemetry just showed redo last status is failing because status file was not found. The scenario you described could also be a reason. Should I change the implementation here to exit with success instead of failure?